docs/mkdocs/labsv2.md

```diff
@@ -0,0 +1,425 @@
+# KMM v2.0 Labs
+
+KMM Labs is a series of examples and use cases of Kernel Module Management Operator running on an Openshift cluster.
+
+
+## Installation
+
+If you want to learn more about KMM and how to install it, you can refer to the [documentation](https://docs.openshift.com/container-platform/4.13/hardware_enablement/kmm-kernel-module-management.html). The documentation provides detailed information about KMM and step-by-step instructions for the installation process.
+
+
+## Driver Toolkit
+
+We could use any image that includes the necessary libraries and dependencies for building a kernel module for our specific version. However, for ease, we can leverage Driver Toolkit (DTK), which is a container image used as a base image for building kernel modules.
+
+When using DTK, we should set which version of the Driver Toolkit [image](https://github.com/openshift/driver-toolkit#finding-the-driver-toolkit-image-url-in-the-payload) we are using to match the exact kernel version running on OpenShift nodes. In our case, we are using DTK with KMM and KMM will automatically determine what DTK digest should be used if the `DTK_AUTO` build argument is specified in the Dockerfile.
+
+We can get further info at DTK [repository](https://github.com/openshift/driver-toolkit#readme).
+
+
+## Building In-Cluster Modules from Sources
+
+In this example, we will demonstrate how to build the kernel module in our cluster using sources from a git repository. For this and next examples it is assumed that a namespace `labsv2` and a ServiceAccount `labsv2sa` allowed to use the privileged SCC have been previously created.
+
+The process involves several steps:
+
+1. Adding a ConfigMap: We will create a ConfigMap to store all the necessary information for the build process, including the image we are using (DTK in this case), the remote git repository containing the sources, and the compiling commands.
+
+2. Creating the Module object: Next, we'll create the Module object itself. It will configure the registry for the resulting image, set up a secret for possible authentication, specify the previously created ConfigMap for use, and define the nodes where the pods should be scheduled.
```

Comment:
```suggestion
2. Creating the Module object: Next, we'll create the Module object itself. It will specify the image to be built and used, a secret for possible authentication, ConfigMap for building process, and the nodes where the pods should be scheduled.
```